hadoop-ozone/dist/src/main/compose/ozonefs/docker-compose.yaml

```diff
@@ -49,21 +49,53 @@ services:
       environment:
          ENSURE_SCM_INITIALIZED: /data/metadata/scm/current/VERSION
       command: ["/opt/hadoop/bin/ozone","scm"]
-   hadoop3:
+   hadoop32:
       image: flokkr/hadoop:3.1.0
```

Comment:
Can we use official hadoop runner image here?

Reply:
This is not hadoop-runner but real hadoop image. We don't use any specific features, so in theory we can switch to use apache/hadoop images. (Note: this is not modified in this patch, flokkr images was used even before).

But for apache/hadoop image we don't have specific tags only one hadoop:2 and hadoop:3. One solution is to create separated tags for all the versions. It would be possible with creating the proposed dedicated repository:

https://issues.apache.org/jira/browse/HADOOP-16092

Reply:
Agree, let's fix that post 0.4.